MozillaSocial/mozilla-social-ios
# Analytics: add discover screen impression

## Summary
* This PR adds an impression for when the `Discover` screen is viewed

## Implementation Details
* Update `DiscoverTracker`, `DiscoverViewModel`: add tracking events for `Discover` tab viewed

## Test Steps
* Build/run, use the app, access the `Discover` tab a few times and make sure events show up at https://debug-ping-preview.firebaseapp.com/pings/moso-ios-test.

## PR Checklist:
- [ ] ~~Self Accessibility Review~~ not applicable

## Screenshots
